script/bootstrap

```diff
@@ -113,14 +113,14 @@ install_packages_brew()
     echo 'Installing pretty tools useful for code contributions...'
 
     # add clang-format for pretty
-    CLANG_FORMAT_VERSION="clang-format version 16"
-    command -v clang-format-16 || (command -v clang-format && (clang-format --version | grep -q "${CLANG_FORMAT_VERSION}")) || {
-        brew install llvm@16
-        sudo ln -s "$(brew --prefix llvm@16)/bin/clang-format" /usr/local/bin/clang-format-16
-        sudo ln -s "$(brew --prefix llvm@16)/bin/clang-tidy" /usr/local/bin/clang-tidy-16
-        sudo ln -s "$(brew --prefix llvm@16)/bin/clang-apply-replacements" /usr/local/bin/clang-apply-replacements-16
-        sudo ln -s "$(brew --prefix llvm@16)/bin/run-clang-tidy" /usr/local/bin/run-clang-tidy-16
-    } || echo 'WARNING: could not install llvm@16, which is useful if you plan to contribute C/C++ code to the OpenThread project.'
+    CLANG_FORMAT_VERSION="clang-format version 19"
+    command -v clang-format-19 || (command -v clang-format && (clang-format --version | grep -q "${CLANG_FORMAT_VERSION}")) || {
+        brew install llvm@19
+        sudo ln -s "$(brew --prefix llvm@19)/bin/clang-format" /usr/local/bin/clang-format-19
+        sudo ln -s "$(brew --prefix llvm@19)/bin/clang-tidy" /usr/local/bin/clang-tidy-19
+        sudo ln -s "$(brew --prefix llvm@19)/bin/clang-apply-replacements" /usr/local/bin/clang-apply-replacements-19
+        sudo ln -s "$(brew --prefix llvm@19)/bin/run-clang-tidy" /usr/local/bin/run-clang-tidy-19
+    } || echo 'WARNING: could not install llvm@19, which is useful if you plan to contribute C/C++ code to the OpenThread project.'
```

Comment:
Can we create a constant for the version to avoid repeating the version changes?